app/models/reservation.rb

```diff
@@ -2,4 +2,6 @@ class Reservation < ApplicationRecord
   belongs_to :car
   belongs_to :city
   belongs_to :user
+
+  validates :date, presence: true
 end
```

Comment:
I think we should add something else here, like start_date for renting and returning_date so we can calculate fees! Let's keep this is optional for this PR, add it in the next!